packages/lesswrong/components/comments/CommentsNewForm.tsx

```diff
@@ -289,8 +314,11 @@ const CommentsNewForm = ({
     if (formDisabledDueToRateLimit) {
       submitBtnProps.disabled = true
     }
-    
-    return <div className={classNames(classes.submit, {[classes.submitMinimalist]: isMinimalist})}>
+
+    return <div className={classNames(classes.submit, {
+      [classes.submitMinimalist]: isMinimalist,
+      [classes.submitQuickTakes]: isQuickTake,
+    })}>
```

Comment:
NBD but perhaps worth changing this in accordance with [this PR](https://github.com/ForumMagnum/ForumMagnum/pull/9118)